lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -89,9 +50,60 @@ def execute(command, options = {})
         command_runner.output
       end
 
+      def file_exists?(filename)
+        File.exist?(filename)
+      end
+
+      def hostname
+        execute('hostname -f')
+      end
+
+      def install_packages(packages, options = {})
+        options.validate_options!(:assumeyes)
+        yum_options = []
+        yum_options << '-y' if options[:assumeyes]
+        execute!("yum #{yum_options.join(' ')} install #{packages.join(' ')}", :interactive => true)
+      end
+
+      def server?
+        find_package('foreman')
```

Comment:
Absolutely!
Added new features. `foreman-server` and `foreman-proxy` to be more specific and `label :foreman` was already taken.